packages/zowe-explorer/package.json

```diff
@@ -2097,12 +2123,10 @@
     "@zowe/zosmf-for-zowe-sdk": "8.0.0-next.202403132009",
     "@zowe/zowe-explorer-api": "3.0.0-next-SNAPSHOT",
     "dayjs": "^1.11.10",
+    "disposablestack": "^1.1.3",
```

Comment:
Should this be a dev dep instead of a prod dep, since it looks like we only use the package in unit tests?